core/arch/x86/mangle.c

```diff
@@ -2162,6 +2166,7 @@ mangle_syscall_arch(dcontext_t *dcontext, instrlist_t *ilist, uint flags, instr_
                                         NULL);
         /* sysenter goes here */
         PRE(ilist, next_instr, post_sysenter);
+        /* XXX i#3307: necessary to instr_set_translation_mangling_epilogue? */
```

Comment:
This one seems like it needs something.  translate_walk_track() claims we'll just fail for syscall mangling but I don't see how that's accomplished, at least for this Mac mangling.

Reply:
Keeping comment / offline discussion. May add or xref in comment